### DISCUSSION TITLE
Setup app on local machine

### DISCUSSION THREAD

**Discussion by jhamadhav** [vigneshshettyin/Bulk-Mailer#3] [Q&A] [answered]
I followed the exact method mentioned in the README file but while installing packages using pip, I encountered two errors :
1. The command for installing packages from *requirements.txt* i.e. "pip install requirements.txt" doesn't work
but it works with the command "pip -r install requirements.txt" 
2. The issue is while installing this is the error that shows up

The error showed is as follows : 
*Using legacy 'setup.py install' for mysqlclient, since package 'wheel' is not installed.
Installing collected packages: mysqlclient, gunicorn, Flask-SQLAlchemy, Flask-Login
Running setup.py install for mysqlclient ... error*

Please suggest some solution.
![bulk-mail-pip-error](https://user-images.githubusercontent.com/37475140/110455082-00dd3180-80ee-11eb-8710-8035a41578f2.png)

**Comment by srisatya12** [comment]
I want to contribute to this issue. Please assign me this task as a part of GSSoC'21

**Comment by vigneshshettyin** [comment]
> I want to contribute to this issue. Please assign me this task as a part of GSSoC'21

Sure!

Let me solve issue of @jhamadhav first. I'll make a short video as in how to set up environment.

**Comment by vigneshshettyin** [comment, marked as answer]
Follow the steps below to setup env for any flask app

First fork the recent repo 

then rest steps in this video https://youtu.be/cNcXo_IROJ0

@jhamadhav

**Reply by jhamadhav** [reply]
Thank you!
